pkg/kube/kube_handler.go

```diff
@@ -63,15 +63,29 @@ type ClusterConfig struct {
 	// TLS requests without requiring the cert authority validation.
 	Insecure bool `json:"insecure"`
 
-	// PinnipedProxy enables user requests to the cluster api server to be
-	// proxied via pinniped for an exchange of the user token for client certs.
-	// This is required on clusters which are not configured with OIDC support.
-	PinnipedProxyURL string `json:"pinnipedProxyURL"`
+	// PinnipedConfig is an optional per-cluster configuration specifying
+	// the pinniped namespace, authenticator type and authenticator name
+	// that should be used for any credential exchange.
+	PinnipedConfig PinnipedConciergeConfig `json:"pinnipedConfig,omitempty"`
+}
+
+// PinnipedConciergeConfig enables each cluster configuration to specify the
+// pinniped-concierge installation to use for any credential exchange.
+type PinnipedConciergeConfig struct {
+	// ExchangeCredentials just flags whether this cluster should use
+	// pinniped to exchange credentials.
+	ExchangeCredentials bool `json:"exchangeCredentials"`
```

Comment:
I would call this var just `Enable`, that way it's self explanatory without Pinniped context